### PR TITLE
rework of densepose alphalab content

### PR DESCRIPTION
**Pull request by mikelgg93**

### CODE REVIEW

File: src/alpha-lab/dense-pose.md [hunk 1]
```diff
@@ -1,145 +1,78 @@
 ---
+title: "Map gaze onto body parts using DensePose"
```
[the remark this answers is not in the document]
Comment (N-M-T) [reply src/alpha-lab/dense-pose.md:2]: Just to standardise with YT video and sidebar title

File: src/alpha-lab/dense-pose.md [hunk 2]
```diff
@@ -1,145 +1,78 @@
...
+## What tools enable this?
 
+Pupil Cloud currently offers a [Face Mapper enrichment](/enrichments/face-mapper/), which tracks faces in scene video and determines whether they were gazed at. However, tracking the rest of the body is currently not possible. This is where our guide comes in. By following our instructions, you will be able to automatically detect and track body parts of people in your eye tracking recordings. Subsequently, you can map gaze onto them, enabling a deeper understanding of how the eye tracker wearer visually interacted with them.
 
-## Running in your browser
+Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information about DensePose, please refer to the paper, [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
```
Comment (N-M-T) [inline src/alpha-lab/dense-pose.md:39]:
```suggestion
Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information about DensePose, please refer to [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
```

File: src/alpha-lab/dense-pose.md [hunk 3]
```diff
@@ -1,145 +1,78 @@
...
-
-#### Video
-
-You've already seen the accompanying video above. There's no mystery – you have a bounding box delimiting each person detected, a blue shaded mask over the body parts, a yellow highlighted body part when it's gazed at, and the typical red circle for the gaze position.
+<div class="mb-4" style="display:flex;justify-content:center;">
+  <v-img class="rounded" :src="require(`../media/alpha-lab/densepose-colab.png`)" width="100%" />
+</div>
 
-#### Gaze map image
+## Contribution
```
Comment (N-M-T) [inline src/alpha-lab/dense-pose.md:57]:
```suggestion
## Results
```

File: src/alpha-lab/dense-pose.md [hunk 4]
```diff
@@ -1,145 +1,78 @@
...
-<!-- This is collapsed   -->
-- Model weights:<br>
-<code>densepose_rcnn_R_50_FPN_DL_s1x</code><br><br>
-- Why is there no gaze recorded on the back of the head, hands or feet?<br>
-There is no definition for those parts in DensePose. Likewise the frontal view of the arms in the picture refers to the inside of the arms, not the front.<br>
-<br>
-</details>
\ No newline at end of file
+::: tip 
+Need assistance in implementing your gaze-contingent task? Reach out to us via email at [info@pupil-labs.com](mailto:info@pupil-labs.com) or on our [Discord server](https://pupil-labs.com/chat/) or visit our [Support Page](https://pupil-labs.com/products/support/) for dedicated support options.
```
Comment (N-M-T) [inline src/alpha-lab/dense-pose.md:77]:
```suggestion
Need assistance implementing your own DensePose + gaze tracking application? Reach out to us via email at [info@pupil-labs.com](mailto:info@pupil-labs.com) or on our [Discord server](https://pupil-labs.com/chat/), or visit our [Support Page](https://pupil-labs.com/products/support/) for dedicated support options.
```

File: src/alpha-lab/dense-pose.md [hunk 5]
```diff
@@ -1,145 +1,78 @@
...
+## What tools enable this?
 
+Pupil Cloud currently offers a [Face Mapper enrichment](/enrichments/face-mapper/), which tracks faces in scene video and determines whether they were gazed at. However, tracking the rest of the body is currently not possible. This is where our guide comes in. By following our instructions, you will be able to automatically detect and track body parts of people in your eye tracking recordings. Subsequently, you can map gaze onto them, enabling a deeper understanding of how the eye tracker wearer visually interacted with them.
 
-## Running in your browser
+Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information about DensePose, please refer to [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
```
Comment (N-M-T) [inline src/alpha-lab/dense-pose.md:39]:
```suggestion
Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information, please refer to [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
```

**Review by N-M-T — APPROVED** [review]

**Comment by N-M-T** [comment]
@willpatera, would you be able to review this and merge if appropriate?

File: src/alpha-lab/dense-pose.md [hunk 6]
```diff
@@ -1,145 +1,78 @@
...
-
-## Outputs
-
-After the code is executed, new files will be generated. Check the new DensePoseColab folder. Let's see what's in those files!
-
-#### Video
-
-You've already seen the accompanying video above. There's no mystery – you have a bounding box delimiting each person detected, a blue shaded mask over the body parts, a yellow highlighted body part when it's gazed at, and the typical red circle for the gaze position.
+<div class="mb-4" style="display:flex;justify-content:center;">
+  <v-img class="rounded" :src="require(`../media/alpha-lab/densepose-colab.png`)" width="100%" />
```
Comment (willpatera) [inline src/alpha-lab/dense-pose.md:54]: This colab screenshot isn't doing much to enhance the article IMO. Can we consider omitting it?

**Review by willpatera — CHANGES_REQUESTED** [review]
This is certainly an improvement. I feel like we're almost there. A couple of small changes would make it even better IMO. Please see my comments.

File: src/alpha-lab/dense-pose.md [hunk 7]
```diff
@@ -1,145 +1,78 @@
...
-<details>
-    <summary>This tool employs <b>DensePose</b> (<a href="https://ai.facebook.com/tools/detectron2/">detectron2</a>, Meta AI). Click here to learn more</summary><br>
-<!-- This is collapsed   -->
-    DensePose is a method for dense human pose estimation and dense human body part segmentation. It's based on the Mask R-CNN architecture and is trained on the COCO dataset. DensePose is now a part of Detectron2's framework. You can read all the details in their paper <a href="https://arxiv.org/abs/1802.00434">DensePose: Dense Human Pose Estimation In The Wild.</a>
-</details>
-<br>
+## Understanding visual behaviour on body parts
 
-## What you'll need
+Understanding which body parts people look at during interactions, whether visual or otherwise, is an important topic in fields ranging from sports science to psycholinguistics. This guide shows you how to use Neon or Pupil Invisible eye tracking with Meta's DensePose to characterise gaze behaviour on body parts that appear in the scene video, as shown above.
```
[the remark this answers is not in the document]
Comment (N-M-T) [reply src/alpha-lab/dense-pose.md:33]: @willpatera, adding the link here is good. I'm not sure what you meant with the quotation mark, however. Should that be a comma, or something else?

File: src/alpha-lab/dense-pose.md [hunk 8]
```diff
@@ -1,145 +1,78 @@
...
+## What tools enable this?
 
+Pupil Cloud currently offers a [Face Mapper enrichment](/enrichments/face-mapper/), which tracks faces in scene video and determines whether they were gazed at. However, tracking the rest of the body is currently not possible. This is where our guide comes in. By following our instructions, you will be able to automatically detect and track body parts of people in your eye tracking recordings. Subsequently, you can map gaze onto them, enabling a deeper understanding of how the eye tracker wearer visually interacted with them.
 
-## Running in your browser
+Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released DensePose, a new tool that promises to solve many of these problems. As a result, we have decided to incorporate DensePose into our guide. For more information, please refer to [DensePose: Dense Human Pose Estimation In The Wild](https://arxiv.org/abs/1802.00434).
```
Comment (N-M-T) [inline src/alpha-lab/dense-pose.md:39]:
```suggestion
Until recently, existing methods for tracking body parts in eye tracking scene video were not very robust or accurate. However, Meta has released [DensePose](https://arxiv.org/abs/1802.00434), a new tool that promises to solve many of these problems. As a result, we have decided to incorporate it into our guide.

Comment (N-M-T) [reply src/alpha-lab/dense-pose.md:39]: @willpatera how about this shortened version?

File: src/alpha-lab/dense-pose.md (hunk 6 above)
Comment (N-M-T) [reply src/alpha-lab/dense-pose.md:54]: I will omit it

**Pull request merged**